src/Utilities/Array.hpp

```diff
@@ -16,8 +16,8 @@
 namespace cpp17 {
 namespace detail {
 template <typename T, size_t Size, size_t... Is>
-// NOLINTNEXTLINE(modernize-avoid-c-arrays)
 constexpr std::array<T, Size> convert_to_array(
+    // NOLINTNEXTLINE(modernize-avoid-c-arrays)
```

Comment:
I am struggling with the same line in PR #2392. This change didn't work for me there. I think we (i.e., whoever merges first) may just have to add a NOLINT to each of these lines...

Reply:
My fix just moved the NOLINTNEXTLINE down a line from where it used to be.  Seems to work for me.  Clang-tidy still fails on other things though (all in a boost routine).

Reply:
👍 What I didn't realize when I wrote my earlier comment, is that I was on a slightly older `develop` where the file had different line breaks. On current develop this change does work for me 🙂